### PR TITLE
Update to runtime 6.3

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

File: org.rncbc.qsynth.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
     "--device=all",
     /* JACK */
     "--filesystem=xdg-run/pipewire-0",
-    "--system-talk-name=org.freedesktop.RealtimeKit1",
```
Comment (pedrolcl) [inline org.rncbc.qsynth.json:18]: Sure? FluidSynth requests realtime priority via D-Bus to RealtimeKit.

Comment (hfiguiere) [reply org.rncbc.qsynth.json:18]: it will never work that's for sure. 

I didn't see any issues either popping out.

Comment (pedrolcl) [reply org.rncbc.qsynth.json:18]: The error message is shown in the "Messages" window:
> fluidsynth: warning: Failed to set thread to high priority

But you are right: it doesn't work anyway. How can we solve it?

Comment (hfiguiere) [reply org.rncbc.qsynth.json:18]: patch fluidsynth to use the (realtime) portal. but then if the host doesn't have an up to date version it still won't work. 

A bit more context on what I did for pipewire: https://www.figuiere.net/hub/wlog/rtkit-portal-pipewire/

Comment (pedrolcl) [reply org.rncbc.qsynth.json:18]: OK, I will open also a PR for the FluidSynth shared module, with another issue.

Comment (hfiguiere) [reply org.rncbc.qsynth.json:18]: It's not the shared module. the work should be done upstream.

**Pull request merged**